report.txt

```diff
@@ -0,0 +1,94 @@
+Number of packages coming from AOSS: 89
+Number of packages coming from public repository: 103
```

Comment:
report will be generated by a script, is it okay to delete this file?

Reply:
We should provide the report as report.txt, i will add it in gitignore so that it is not visible, but user can generate it whenever they want.